wfe/wfe.go

```diff
@@ -1046,11 +1047,21 @@ func (wfe *WebFrontEndImpl) verifyOrder(order *core.Order) *acme.ProblemDetails
 	if len(idents) == 0 {
 		return acme.MalformedProblem("Order did not specify any identifiers")
 	}
-	// Check that all of the identifiers in the new-order are DNS type
+	// Check that all of the identifiers in the new-order are DNS or IPaddress type
+	// Validity check of ipaddresses are done here.
 	for _, ident := range idents {
+		if ident.Type == acme.IdentifierIP {
+			ip := net.ParseIP(ident.Value)
```

Comment:
This and line below can be condensed into `if ip := net.ParseIP(ident.Value); ip == nil {...}`.